### PR TITLE
Update travis to test newer versions of node

### PR DESCRIPTION
**Pull request by LionOps**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ language: node_js
 
 node_js:
   - '0.10'
+  - '0.12'
+  - '4.2'
```
Comment (beck) [inline .travis.yml]: any reason why use the minor version instead of just `4` and `5`?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by beck** [comment]
:+1:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by LionOps** [comment]
Force pushed to squash the two commits.

**Pull request merged**